src/App.tsx

```diff
@@ -1,15 +1,16 @@
-import React from "react";
+import CssBaseline from "@mui/material/CssBaseline";
```

Comment:
same here as below comment

Reply:
removed this, based on our conversation you will be working on this.